doc/source/train/user_guide.rst

```diff
@@ -120,6 +120,11 @@ training.
         -       X = X.to_device(device)
         -       y = y.to_device(device)
 
+    .. note::
+       You can choose not to use the ``prepare_model`` or ``prepare_data_loader``. However, if you choose not to
+       do so, then you are responsible for moving the model & data to the right devices, adding ``DistributedSampler``
+       to your ``DataLoader`` and wrapping the model in ``DistributedDataParallel`` in your training function.
+
```

Comment:
I'm a little wary about how this reads to the end user. Maybe instead we can just update each of the steps above to tell the user they can _either_ use the utility or implement the logic themselves?

Reply:
Updated, take another look?

Reply:
TY this feels more straightforward now